scss/components/tooltip.scss

```diff
@@ -0,0 +1,82 @@
+@import "../core/settings";
+@import "../core/mixins";
+@import "../core/functions";
+
+/*!
+.tn-tooltip
+    .tn-tooltip__arrow
+    .tn-tooltip__content
+*/
+$block: ns(tooltip);
+
+.#{$block} {
+
+    //LOCAL VARS (set all themeable properties, always include !default)
+    $tn-tooltip-content-background-color: tn-color(text, 3) !default;
+    $tn-tooltip-content-color: tn-color(text-inverse, 1) !default;
+    $tn-tooltip-padding: tn-space(s) !default;
+    $tn-tooltip-arrow-offset: (tn-space(s)) - 2 !default ;
+    $tn-tooltip-arrow-width: $tn-tooltip-arrow-offset !default;
+
+    //BLOCK BASE *******************************************
+    @include reset;
+    background: $tn-tooltip-content-background-color;
+    padding: $tn-tooltip-padding;
+    display: inline-block;
+    position: relative;
+
+    //ELEMENTS *******************************************
+    &__arrow{
```

Comment:
Did you try to do this with `:before` pseudo elements? May be more trouble than it's worth but it would reduce the markup considerably.